### DISCUSSION TITLE
Control Gpio through a command "/..."

### DISCUSSION THREAD

**Discussion by RomPe59** [cameroncros/OctoPrint-DiscordRemote#198] [General]
Hello everyone ! 
First of all, thank you a lot for this plugin which is very useful 😄 
I was wondering if it was possible to control GPIO directly through a command in order to remotely switch on/off the light for example.
If it's not possible, then it would be a good improvement for the app to add something to support this i think !
What do you think about that ? 💭

**Comment by cameroncros** [comment]
There is no reason that couldnt be implemented, but I am not making any new fixes to this plugin at the moment. If you do implement something, I would be happy to merge it. I can also give assistance if you need guidance for making this change.

**Reply by RomPe59** [reply]
Hello @cameroncros,
Sadly, i'm not good enough to create something, it's too complex for me and i don't have those skills 😭 
But thank you for your quick answer, I'm going to try the PSU solution for the moment !

**Comment by cameroncros** [comment]
One option you do have is to abuse the PSU Control plugin and use that to toggle a single GPIO pin on and off:

https://plugins.octoprint.org/plugins/psucontrol/
If you have that installed you can use `/poweron` and `/poweroff` to turn on the GPIO pin.
